.github/workflows/ci.yml

```
@@ -10,142 +7,105 @@ on:
-        path: releng/ide-dmg-builder/Espressif-IDE-macosx-cocoa-x86_64.dmg
-
-    - name: Upload espressif-ide-macosx.cocoa.aarch64 dmg
-      if: ${{ !cancelled() }}
-      uses: actions/upload-artifact@v4
-      with:
-        name: Espressif-IDE-${{ env.VERSION }}-macosx.cocoa.aarch64
-        path: releng/ide-dmg-builder/Espressif-IDE-macosx-cocoa-aarch64.dmg
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)